src/Routes.js

```diff
@@ -1,20 +1,25 @@
 import React from 'react';
 import { Route, Switch, BrowserRouter } from 'react-router-dom';
+import styled from 'styled-components';
 import Careers from './containers/Careers.js';
 import LikedCareers from './containers/LikedCareers.js';
 import CareerDetails from './containers/CareerDetails.js';
 import NavBar from './components/NavBar';
 
+const AppWrapper = styled.div`
+  overflow: hidden;
+`;
```

Comment:
Just a question - what does the wrapper `overflow:hidden` do?

Reply:
It was to hide the cards after they had been thrown out the stack

Reply:
:+1: